### PR TITLE
Next up display for skins

### PR DESCRIPTION
**Pull request by madikarizma**
* Add next up display styles for each skin
* Fix alignment issues per skin

JW7-3137

### CODE REVIEW

File: src/css/imports/nextup.less [hunk 1]
```diff
@@ -5,15 +5,14 @@
   -webkit-font-smoothing: antialiased;
   -moz-font-smoothing: antialiased;
   background-color: transparent;
-  bottom: calc(@controlbar-height + 0.5em);
```
Comment (robwalch) [inline src/css/imports/nextup.less]: So there's is no default bottom w/o a skin? You should put most seven skin settings here and only overwrite those needed in other skins. Skin authors shouldn't be required to include every rule.

File: src/css/skins/seven.less [hunk 2]
```diff
@@ -54,4 +64,38 @@
...
+    background: @nextup-body-background;
+  }
+  .jw-nextup-thumbnail-visible + .jw-nextup-title::after {
+    background: linear-gradient(
+      -180deg,
+      fade(@nextup-title-overflow-background, 0%) 0%,
+      @nextup-title-overflow-background 100%
+    );
+  }
+  .jw-nextup-title {
```
Comment (johnBartos) [inline src/css/skins/seven.less]: Add these coloring styles to the `basic-skins-style` mixin in `mixins.less`

Comment (madikarizma) [inline src/css/skins/seven.less]: Something I need to do is get more familiar with how this code base works, but this and @robwalch's comment helped and the changes I made should be closer to what you had in mind.

Comment (johnBartos) [inline src/css/skins/seven.less]: Looks like you got it right, going to test it locally & merge

**Pull request merged**